bazel/repositories.bzl

```diff
@@ -477,7 +477,30 @@ def _com_github_facebook_zstd():
 
 def _com_google_cel_cpp():
     external_http_archive(
-        "com_google_cel_cpp",
+        name = "com_google_cel_cpp",
+        patch_args = ["-p1"],
+        patches = ["@envoy//bazel/foreign_cc:cel-cpp.patch"],
+    )
+    external_http_archive(
```

Comment:
This used to be encapsulated via https://github.com/google/cel-cpp/blob/master/bazel/deps.bzl#L82. Does that no longer work?